pom.client.xml

```diff
@@ -684,31 +712,69 @@
       </reporting>
     </profile>
 
-    <!-- Setup for Java 9+ -->
+    <!-- Setup for Java 11+ -->
     <profile>
       <id>java-lts</id>
```

Comment:
Should we change the profile id to java-11?

Reply:
We could do. In a few years we would need to rename this to java-17 when that becomes the LTS version, which is why I've hesitated so far.

Reply:
Given we aren't really using it by name any longer I guess it doesn't really matter.